src/types.cairo

```diff
@@ -195,10 +195,12 @@ pub struct QuoteTokenInfo {
 //
 
 pub mod pragma {
-    #[derive(Copy, Drop, Serde)]
+    #[derive(Copy, Drop, PartialEq, Serde, starknet::Store)]
     pub enum AggregationMode {
+        #[default]
```

Comment:
Defaults to Median mode. This is necessary for reading the settings of an address that has not been added.

Reply:
> reading the settings of an address that has not been added

That would only happen by mistake right? E.g. calling `fetch_price` with a wrong yang address in which case the fn would fail with the assert we have in place.

Reply:
It was actually caught by `test_update_prices_fails_with_no_yangs_in_seer` in the Seer test suite for calling `update_prices` without any yangs, so it is indeed quite an edge case. Should I remove the default?

Reply:
Nah, keep it, it's cool, I was just curious.